dandiapi/urls.py

```diff
@@ -163,6 +165,16 @@ def to_url(self, value):
     ),
 ]
 
+if settings.DANDI_ENABLE_SITEMAP_XML:
+    urlpatterns += [
+        path(
+            'sitemap.xml',
+            sitemap,
+            {'sitemaps': sitemaps},
+            name='django.contrib.sitemaps.views.sitemap',
+        ),
+    ]
```

Comment:
Forgot to mention it in my self-review, but is there any need to cache this view?

My guess is there's no need, at least not at the scale we're currently running at. Apparently, past 50000 entries, you have to split your sitemap.xml file up or risk Google ignoring you, but of course we're nowhere near that number.

Reply:
I don't see a need, not at the moment anyway.